homeassistant/components/frontend/__init__.py

```diff
@@ -448,7 +490,9 @@ def get(self, request, extra=None):
         if panel == 'states':
             panel_url = ''
         else:
-            panel_url = hass.data[DATA_PANELS][panel].webcomponent_url
+            panel_url = hass.data[DATA_PANELS][panel].webcomponent_url_latest \
```

Comment:
Can we flatten this inline if…else into the parent if…else

Reply:
Done